### DISCUSSION TITLE
How to avoid "The file has been changed by another application." with Cryptomator via WebDAV on macOS

### DISCUSSION THREAD

**Discussion by dannys42** [iterate-ch/cyberduck#15368] [Q&A]
I'm attempting to use a Numbers document stored in a Cryptomator vault on a WebDAV drive.  Everything works, except that if I wait more than ~5 seconds between saves, I get a warning "could not be saved. The file has been changed by another application."  I can get by with selecting "Save Anyway", but wondering if there are any good fixes to this problem?

I've tried doing the same thing with a Cryptomator vault on a local directory as well as an unencrypted WebDAV drive without Cryptomator.  In both cases, I don't run into this issue.

**Comment by dannys42** [comment]
FYI, testing the same Cryptomator vault over FTP showed no issues as well.  I'll likely just use this, but leaving the question in case it's something that can/should be fixed?

**Comment by dkocher** [comment]
My guess here is that we fail to save custom attributes for the file on WebDAV that includes the modification date. Can you let us know the WebDAV storage server you are connecting to?

**Reply by dannys42** [reply]
I'm using WebDAV on QNAP, which appears to be apache.

Server version: Apache/2.4.55 (Unix)
Server built:   Sep 26 2023 04:45:57

**Comment by dannys42** [comment]
Just adding that I switch to FTP-SSL and I'm still seeing the same problem. :-(
